Apps/WebClient/src/NewClientApp/src/components/private/dependent/AddDependentComponent.vue

```diff
@@ -0,0 +1,457 @@
+                    <v-alert
+                        v-if="isError"
+                        data-testid="dependentErrorBanner"
+                        class="d-print-none mx-4 mt-4"
+                        type="error"
+                        variant="outlined"
+                        border
+                    >
+                        <template #text>
+                            <p
+                                data-testid="dependentErrorText"
+                                class="text-body-1"
+                            >
+                                <span v-if="isErrorDataMismatch">
+                                    The information you entered does not match
+                                    our records. Please try again with the exact
+                                    given and last names on the BC Services
+                                    Card.
+                                </span>
+                                <span v-else-if="isErrorNoHdid">
+                                    Please ensure you are using a current
+                                    <a
+                                        href="https://www2.gov.bc.ca/gov/content/governments/government-id/bc-services-card"
+                                        target="_blank"
+                                        rel="noopener"
+                                        >BC Services Card</a
+                                    >.
+                                </span>
+                                <span v-else-if="isErrorProtected">
+                                    Unable to add dependent.
+                                </span>
+                                <span v-else>{{ errorMessage }}</span>
+                            </p>
+                            <span
+                                v-if="isErrorProtected"
+                                data-testid="condensed-error-contact-message"
+                                class="text-body-1"
+                            >
+                                Please contact
+                                <a href="mailto:HealthGateway@gov.bc.ca"
+                                    >HealthGateway@gov.bc.ca</a
+                                >.
+                            </span>
+                            <span
+                                v-else
+                                data-testid="not-condensed-error-contact-message"
+                                class="text-body-1"
+                            >
+                                If you continue to have issues, please contact
+                                <a href="mailto:HealthGateway@gov.bc.ca"
+                                    >HealthGateway@gov.bc.ca</a
+                                >.
+                            </span>
+                        </template>
+                    </v-alert>
+                    <v-container data-testid="newDependentModalText">
```

Comment:
this testid should be kebab-case for consistenc

I don't really think the v-container is necessary. If it helped with the layout though feel free to leave it.

Reply:
Removed v-container.